indigo/distribution.yaml

```diff
@@ -10,6 +10,21 @@ release_platforms:
   - saucy
   - trusty
 repositories:
+  acado:
+    doc:
+      type: git
+      url: https://github.com/clearpath-gbp/acado-release.git
+      version: release/indigo/acado
+    release:
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/clearpath-gbp/acado-release.git
+      version: 1.2.1-1
+    source:
+      type: git
+      url: https://github.com/clearpath-gbp/acado-release.git
+      version: release/indigo/acado
```

Comment:
Okay, I pushed the branch to a name without slashes, hopefully that's acceptable.